README.md

```diff
@@ -141,14 +135,14 @@ print(app.myButton["text"]) # outputs text in the button 'Click me'
 app.mainloop()
 ```
 
->Note: Its advisable that you use widget names that are valid python identifiers to avoid 
->possible issues while use the dot syntax to access the widget from the builder object.
+>Note: Its advisable that widget names are not named as reserved Python keywords to avoid 
```

Comment:
maybe remove the "named", seems redundant. Also, you need to mention something about being a valid python identifier. This allows us to cover other aspects such as the need to start with a letter or underscore, not having special characters and not being a reserved keyword.